pytensor/link/jax/dispatch/scan.py

```diff
@@ -29,16 +40,13 @@ def scan(*outer_inputs):
         n_steps = scan_args.n_steps
         seqs = scan_args.outer_in_seqs
 
-        # TODO: mit_mots
         mit_mot_in_slices = []
+        if scan_args.outer_in_mit_mot:
+            raise NotImplementedError("JAX Scan with MIT-MOT not supported yet.")
 
         mit_sot_in_slices = []
         for tap, seq in zip(scan_args.mit_sot_in_slices, scan_args.outer_in_mit_sot):
-            neg_taps = [abs(t) for t in tap if t < 0]
```

Comment:
AFAICT there is no such thing as positive taps

Reply:
Those don't make it into Scan as taps. If you set something like `sequences={"value": seq, taps=[1, 2]}` (I don't remember the exact syntax), you will get two sequence inputs `[seq[1:-1], seq[2:]` or something like that. Internally sequences never have taps